app/org/sagebionetworks/bridge/dynamodb/DynamoStudy.java

```diff
@@ -187,6 +189,30 @@ public void setUserProfileAttributes(Set<String> profileAttributes) {
     }
 
     /** {@inheritDoc} */
+    @DynamoDBMarshalling(marshallerClass = StringSetMarshaller.class)
```

Comment:
Do we really need the StringSetMarshaller? I know for sure that DDB natively supports StringSets. I've seen code examples of DDB mappers using StringSets without needing any custom marshallers.